src/main/java/com/wire/bots/roman/Sender.java

```diff
@@ -69,8 +71,7 @@ private UUID send(IncomingMessage message, WireClient client) throws Exception {
                 break;
             }
             case "call": {
-                final String content = "{\"version\":\"3.0\",\"type\":\"GROUPSTART\",\"sessid\":\"\",\"resp\":false}";
-                final Calling calling = new Calling(content);
+                final Calling calling = new Calling(mapper.writeValueAsString(message.call));
```

Comment:
this is a breaking change, is there any other bot (then admin broadcast) that use this type?

Reply:
I believe it is :)

Reply:
if it's just broadcast then it is fine, otherwise it would be better to check something like that:
```java
String content = message.call != null ?  mapper.writeValueAsString(message.call) : "some default";
```